addons/rmsmartshape/plugin.gd

```diff
@@ -824,38 +826,40 @@ func draw_mode_edit_vert(overlay: Control, show_vert_handles: bool = true) -> vo
 	var t: Transform2D = get_et() * shape.get_global_transform()
 	var verts: PackedVector2Array = shape.get_vertices()
 	var points: PackedVector2Array = shape.get_tessellated_points()
-	draw_shape_outline(overlay, t, points, shape.modulate)
+	var color := Color(1, 1, 1, 0.6)
```

Comment:
Make this color a constant to make it reusable and to prevent needless allocations.